util/ranger/ranger_test.go

```diff
@@ -627,8 +636,11 @@ func (s *testRangerSuite) TestIndexRangeForUnsignedInt(c *C) {
 
 func (s *testRangerSuite) TestColumnRange(c *C) {
 	defer testleak.AfterTest(c)()
-	store, err := newStoreWithBootstrap(c)
-	defer store.Close()
+	dom, store, err := newDomainStoreWithBootstrap(c)
```

Comment:
maybe we can implement function `TearDownTest` for `testRangerSuite` and put the code deferred into that function.

Reply:
This will modify too much, this is a quick fix...so may we do it later~?

Reply:
ok